parsl/executors/workqueue/executor.py

```diff
@@ -167,25 +171,25 @@ class WorkQueueExecutor(NoStatusHandlingExecutor):
     """
 
     def __init__(self,
-                 label="WorkQueueExecutor",
```

Comment:
put `@typeguard.typechecked` on the front of this and the type annotations will be checked at runtime, as users pass in correct/incorrect values.

Reply:
Added in 111b18246d8085446aea49291497db5bb0e45dfa